src/client/Microsoft.Identity.Client/AppConfig/ExternalTokenResult.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class ExternalTokenResult
+    {
+        public string[] scopes { get; set; } // (or resource?) 
```

Comment:
Actually AAD does not give us back scopes for this flow. We just use the input scope to store data in the cache.